src/ace_header.F90

```diff
@@ -144,6 +145,10 @@ module ace_header
     integer                :: urr_inelastic
     type(UrrData), pointer :: urr_data => null()
 
+    ! Multipole data
+    logical                       :: mp_present
+    type(MultipoleArray), pointer :: multipole => null()
```

Comment:
Is there a specific reason you need this to be `pointer`? It is highly preferable in F2003+ to use `allocatable` rather than `pointer`.